src/Orleans/SystemTargetInterfaces/IReminderTable.cs

```diff
@@ -15,7 +14,7 @@ namespace Orleans
     /// </summary>  
     public interface IReminderTable
     {
-        Task Init(GlobalConfiguration config, Logger logger);
+        Task Init(GlobalConfiguration config);
```

Comment:
removed Logger from IReminderTable.Init . now it gets logger from DI in constructor injection